playground/node-compat/worker-basic/wrangler.toml

```diff
@@ -1,2 +1,4 @@
+name = "worker"
```

Comment:
What's the reason for changing all these worker names? Does it matter?

Reply:
mh... probably not... I mean... it can make things clearer if we were to ever deploy any of these... 🤷

it is a pretty minimal thing indeed... but also I don't see why not having more specific names 🤷

not a big deal, totally up to you, if you want to keep it to `worker` that's ok by me 🙂

Reply:
It's fine, I'm happy to do it. I just asked because the worker names haven't changed in this PR.

Reply:
ah.... sorry I missed that 😅

well again, whatever you prefer I'll be fine with 🙂👍